datalad/core/local/run.py

```diff
@@ -174,7 +174,17 @@ class Run(Interface):
              code_cmd="""\
              datalad run -m 'run my script' -i 'data/*' \\
              -i 'datafile.txt' -o 'output_dir/*' -o \\
-             'outfile.txt' 'code/script.sh'""")
+             'outfile.txt' 'code/script.sh'"""),
+        dict(test="Use ** to match any file at any directory depth recursively. "
```

Comment:
```suggestion
        dict(text="Use ** to match any file at any directory depth recursively. "
```

I believe this is the cause for the current failures in the CI

Reply:
Oops! Thanks for spotting this.